### DISCUSSION TITLE
I have a problem with q-selector.

### DISCUSSION THREAD

**Discussion by hoangphuckt7** [quasarframework/quasar#13637] [General - Components / Directives / etc]
![image](https://user-images.githubusercontent.com/81405523/172587832-94c323bc-992e-48b1-b5f6-a3d60fcd1ad9.png)
![image](https://user-images.githubusercontent.com/81405523/172587985-64965345-badc-4296-a3a5-c7327b0d6681.png)
Please help me.

**Comment by MilosPaunovic** [comment]
You don't say what is the problem. Code reproduction on Codepen is always a good start.
